course/lesson_00_the_setup/README.md

```diff
@@ -2,10 +2,23 @@
 #### introduction
 - Install [Python 3.6](https://www.python.org/downloads/release/python-364/),
 - Install [PyCharm Community Edition IDE](https://www.jetbrains.com/pycharm/download/) - strongly recommended,
+- `pip install bpython ipython pytest ipdb`,
+<!--  How does pip work on Windows? -->
 - Install [git download](https://git-scm.com/downloads),
 - Introductory YouTube video for git,
 
 #### Practice
-1. Fork this repository
-2. Create your directory in  /students/(surname).(name)
-3. ...
+1. Install Python and PyCharm,
+1. Fork the repository,
+1. Git clone your fork,
+1. Create your directory in  /students/(surname).(name),
+1. Copy `lesson_00_the_setup` to your directory,
+1. Example code,
+    1. Paste bad code,
+    1. Commit,
+    1. Fix what the IDE suggests to fix,
+    1. Commit,
+    1. Run the tests `py.test`
```

Comment:
poprawianie stylu i odpalanie testów przed hello worldem to IMO za mocne wejście :) wystarczyłoby żeby skomitowali nawet pusty plik, zdążą się napoprawiać stylu

Reply:
Potrzeba czegoś, żeby poćwiczyli te trzy commity. Myślałem, żeby ich oswoić z koncepcją, że nie muszą wszystkiego rozumieć. Jakieś inne pomysły?

Reply:
Mi się tez wydaje, że w ramach ćwiczenia z gitem wystarczy skomitować nie pusty plik tekstowy.